.github/workflows/update-whats-new-ids.yml

```diff
@@ -19,8 +19,7 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v2
         with:
-          ref: develop
-          persist-credentials: false
+          ref: ${{github.head_ref}}
```

Comment:
interesting. so no `fetch-depth: '0'` but we add that ref. 

i'm curious why we don't need `persist-credentials`

Reply:
the previous workflow had a ref of develop, i thought i could remove the ref and let it choose whichever branch triggered the PR but it turned out to do odd things. but it seems adding a ref specifying the head has solved it!

re: persist credentials, this persists github credentials throughout the workflow i guess? and removes them once it's finished. you can set it to false as we did before to opt out of that. i havent seen this is other checkout actions we've used so i didn't bother with it, but i can add it back

Reply:
ah okay. cool cool. i just wasn't sure if that would affect the whole removing branch protection steps on the `daily-release/*` branch